app/src/main/java/org/schabi/newpipe/settings/BackupRestoreSettingsFragment.java

```diff
@@ -157,6 +202,36 @@ private void requestImportPathResult(final ActivityResult result) {
         }
     }
 
+    private void requestExportSubsResult(final ActivityResult result) {
+        if (result.getResultCode() == Activity.RESULT_OK && result.getData() != null) {
+            final Uri fileUri = result.getData().getData();
+            if (fileUri != null) {
+                final Intent intent = new Intent(
+                        requireContext(), SubscriptionsExportService.class);
+                intent.putExtra(SubscriptionsExportService.KEY_FILE_PATH, fileUri);
+                requireContext().startService(intent);
+                Toast.makeText(requireContext(), R.string.exporting, Toast.LENGTH_SHORT).show();
+            }
+        }
+    }
+
+    private void requestImportSubsResult(final ActivityResult result) {
+        if (result.getResultCode() == Activity.RESULT_OK && result.getData() != null) {
+            final Uri fileUri = result.getData().getData();
+            if (fileUri != null) {
+                final Intent intent = new Intent(
+                        requireContext(), SubscriptionsImportService.class);
+                intent.putExtra(
+                        SubscriptionsImportService.KEY_MODE,
+                        SubscriptionsImportService.PREVIOUS_EXPORT_MODE
+                );
+                intent.putExtra(SubscriptionsImportService.KEY_VALUE, fileUri);
+                requireContext().startService(intent);
+                Toast.makeText(requireContext(), R.string.importing, Toast.LENGTH_SHORT).show();
```

Comment:
This `Toast` is redundant with `showToast(R.string.export_ongoing);` in `SubscriptionsExportService` line 112, and should be removed